staging/src/k8s.io/apimachinery/pkg/runtime/converter_test.go

```diff
@@ -751,6 +752,10 @@ func TestUnrecognized(t *testing.T) {
 			data: "{\"ff\":[\"abc\"],\"fg\":[123],\"fh\":[true,false]}",
 			obj:  &F{},
 		},
+		{
+			data: "{\"fj\":\"\"}",
+			obj:  &F{},
+		},
```

Comment:
I'm not convinced this is good enough? Did that test fail to pass without the change?

Reply:
> I'm not convinced this is good enough? Did that test fail to pass without the change?

This test will fails without the change, The error message is as follows:
```
    converter_test.go:691: Object changed, diff:   &runtime_test.F{
          	... // 7 identical fields
          	H: nil,
          	I: nil,
        - 	J: []uint8{},
        + 	J: nil,
          }
```